lib/grammers-client/src/types/media.rs

```diff
@@ -26,9 +33,48 @@ pub struct Uploaded {
 #[non_exhaustive]
 pub enum Media {
     Photo(Photo),
+    Document(Document),
     Uploaded(Uploaded),
 }
 
+impl Document {
+    pub(crate) fn from_media(
+        document: tl::types::MessageMediaDocument,
+        client: ClientHandle,
+    ) -> Self {
+        Self { document, client }
+    }
+
+    pub async fn download(&mut self, path: &Path) {
```

Comment:
Missing documentation here, please indicate that this is nothing but a shortcut for `ClientHandle::download_media_at_location`, along with the appropriated intra-doc links.